GetARock/GetARock/Screen/Gathering/GatheringListViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  GatheringListViewController.swift
+//  GetARock
+//
+//  Created by Somin Park on 2022/11/23.
+//
+
+import UIKit
+
+enum GatheringListType: CaseIterable {
+    case gatheringCreated
+    case gatheringJoined
+    
+    func toKorean() -> String {
+        switch self {
+        case .gatheringCreated: return "내가 만든 모여락"
+        case .gatheringJoined: return "댓글 단 모여락"
+        }
+    }
+}
+
+class GatheringListViewController: UIViewController {
+    
+    // MARK: - View
+    
+    private let segmentedControlButtons = ViewSwitchedSegmentedControl(buttonTitles:
+                                                                        [GatheringListType.gatheringCreated.toKorean(),
+                                                                         GatheringListType.gatheringJoined.toKorean()])
+    private let gatheringListContentViewController: GatheringCreatedViewController =
+    UIStoryboard(name: "GatheringView", bundle: nil).instantiateViewController(withIdentifier: GatheringCreatedViewController.className) as?
+    GatheringCreatedViewController ?? GatheringCreatedViewController()
```

Comment:
```suggestion
    private let segmentedControlButtons = ViewSwitchedSegmentedControl(
        buttonTitles: [
            GatheringListType.gatheringCreated.toKorean(),
            GatheringListType.gatheringJoined.toKorean()
        ]
    )
    private let gatheringListContentViewController = UIStoryboard(
        name: "GatheringView",
        bundle: nil
    ).instantiateViewController(
        withIdentifier: GatheringCreatedViewController.className
    ) as? GatheringCreatedViewController ?? GatheringCreatedViewController()
```
혹시 이건 어떨까요? 적당한 개행 어렵네요 ㅠㅠ

Reply:
@skycat0212 제가 처음에 했던 개행보다 훨씬 가시성이 좋은 것 같아요! 수정했습니다 감사합니다! bd2b7fd5380d10d4cbb2f12213a5c4a755c4d76b